### DISCUSSION TITLE
Log4j2 vulnerability

### DISCUSSION THREAD

**Discussion by platoxie** [keycloak/keycloak#9122] [Q&A] [answered]
Log4j2 was found to have a jdni vulnerability. Does this affect keycloak（version:14 / 15 / 16）? What is the logging framework used by keycloak by default? Tks！

**Comment by DGuhr** [comment, marked as answer]
Please take a look here: https://github.com/keycloak/keycloak/discussions/9078 - that's our discussion around the topic. If you have any questions left, please ask them there, so we have one place to gather all information. Thank you.
